ui/components/app/srp-input/srp-input.js

```diff
@@ -232,4 +235,8 @@ SrpInput.propTypes = {
    * Otherwise, this is called with an empty string.
    */
   onChange: PropTypes.func.isRequired,
+  /**
+   * Text to show on the left of the Dropdown component. Wrapped in Typography component.
+   */
+  srpText: PropTypes.string,
```

Comment:
This should probably be a required prop. The render method doesn't seem to accommodate this being omitted very well - it still renders the `label` and `Typography` elements but with no contents. Plus I don't see a legitimate use case for omitting it.

Reply:
Making this required would also require adding this to each render call in the srp input unit tests

Reply:
@Gudahtt Done [here](https://github.com/MetaMask/metamask-extension/pull/14425/commits/b08521f00fbcc0ca018296084d50fbfe951269fa)